packages/fuel-gauge/src/contract.test.ts

```diff
@@ -667,6 +680,97 @@ describe('Contract', () => {
     expect(finalBalance).toBe(initialBalance + amountToContract.toNumber());
   });
 
+  it('should transfer asset to a deployed contract just fine (NON-NATIVE ASSET)', async () => {
+    using launched = await launchTestNode({
+      contractsConfigs,
+    });
+    const {
+      provider,
+      wallets: [wallet],
+      contracts: [contract],
+    } = launched;
+
+    const initialBalance = new BN(await contract.getBalance(provider.getBaseAssetId())).toNumber();
+    const amountToContract = bn(10_000);
+    const assetId = TestAssetId.A.value;
+
+    const tx = await wallet.transferToContract(contract.id, amountToContract, assetId);
```

Comment:
Should this be testing batch transfer?

Reply:
@danielbate I've added one additional test case for `transferToContract` since its existent test case was called: `should transfer asset to a deployed contract just fine (NATIVE ASSET)`.

Therefore I've added the: `should transfer asset to a deployed contract just fine (NON-NATIVE ASSET)`